samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -0,0 +1,677 @@
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct phy <phy> <BIG index> <subgroup index>
+
+The supported PHY values are:
+
+  * 1 - 1M
+  * 2 - 2M
+  * 4 - Coded
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct phy 2 0 0
+
+This will set the PHY for subgroup 0 in BIG0 to 2M.
+
+framing - Set framing
+=====================
+
+The command sets the framing for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct framing <unframed/framed> <BIG index> <subgroup index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct framing unframed 0 0
+
+This will set the framing for subgroup 0 in BIG0 to unframed.
+
+rtn - Set number of re-transmits
+================================
+
+The command sets the number of re-transmits for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct rtn <number> <BIG index> <subgroup index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct rtn 2 0 0
+
+This will set the number of re-transmits for subgroup 0 in BIG0 to 2.
+
+sdu - Set SDU
+=============
+
+The command sets the SDU size in octets for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct sdu <octets> <BIG index> <subgroup index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct sdu 60 0 0
+
+This will set the SDU size for subgroup 0 in BIG0 to 60 octets.
+Note that this will not change the bitrate, only the size of the SDU.
+
+mtl - Set Max Transport latency
+===============================
+
+The command sets the Max Transport Latency for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct mtl <ms> <BIG index> <subgroup index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct mtl 10 0 0
+
+This will set the Max Transport Latency for subgroup 0 in BIG0 to 10 ms.
+
+frame_interval - Set frame interval
+===================================
+
+The command sets the frame interval for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct frame_interval <us> <BIG index> <subgroup index>
+
+The supported values are:
+  * 7500
+  * 10000
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct frame_interval 10000 0 0
+
+This will set the frame interval for subgroup 0 in BIG0 to 10 ms (10000 us).
+
+pd - Set presentation delay
+===========================
+
+The command sets the presentation delay for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct pd <us> <BIG index> <subgroup index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct pd 40000 0 0
+
+This will set the presentation delay for subgroup 0 in BIG0 to 40 ms (40000 us).
+
+
+.. _broadcast_configuration_tool_configuration:
+
+Configuration
+*************
+
+The application is pre-configured with a default configuration that is used when the application is started.
```

Comment:
Is this true? When I run `bct start` without setting up any streams it doesn't look like it's broadcasting anything

Reply:
It is regarding the CONFIGs, I can make that more clear